compiler/backend/extccomp.nim

```diff
@@ -580,17 +580,16 @@ proc getCompileCFileCmd*(conf: ConfigRef; cfile: Cfile,
     options.add ' '
     options.add cfile.customArgs
 
-  var compilePattern: string
+  let compilePattern =
+    if noAbsolutePaths(conf): exe
```

Comment:
Just confirming my understand: this is in fact the crux of the change because regardless of flags the pattern is configured in the same fashion?

Reply:
`getCompilerExe(conf, c, cfile.cname)` is a fallback, but it's currently forced in `--genScript:on` mode